corehq/ex-submodules/casexml/apps/phone/tests/dummy.py

```diff
@@ -1,33 +1,50 @@
-from datetime import datetime
-from casexml.apps.phone.models import User
+from mock import MagicMock
+from datetime import date, datetime
 
-DUMMY_USERNAME = "mclovin"
 DUMMY_ID = "foo"
+DUMMY_USERNAME = "mclovin"
 DUMMY_PASSWORD = "changeme"
 
 
 def dummy_user():
-    return User(user_id=DUMMY_ID,
-                username=DUMMY_USERNAME,
-                password=DUMMY_PASSWORD,
-                date_joined=datetime(2011, 6, 9),
-                user_data={"something": "arbitrary"})
+    return MagicMock(
+        username='mclovin',
+        password='guest',
+        user_id='foo',
```

Comment:
this commit is just a WIP, when it's finalized i'll revert back to constants. was trying to remove the function completely